etc/generate_tiles.py

```diff
@@ -216,17 +222,21 @@ def slice_map(thread_count, current_thread):
         tile_dir = os.environ['MAPNIK_TILE_DIR']
     except KeyError:
         tile_dir = "/var/lib/mod_tile/"
+    try:
+        small_map = os.environ["SMALL_MAP"]
```

Comment:
Hah, I'm actually very glad that you added this bikeshed, because I didn't have many good ideas about what to call this environment variable and wasn't entirely happy with what I came up with. I'll adopt something like it.